storage/google_drive/drive_client_wrapper.py

```diff
@@ -195,6 +195,12 @@ def _create_file(source_file_path, target_folder_id, target_file_name=None):
              f"'{source_file_path}' - done. File id is '{file.get('id')}'")
 
 
+def _delete_file(file_id):
+    log.info(f"Deleting file '{file_id}'...")
+    _drive_service.files().delete(fileId=file_id).execute()
```

Comment:
lets closely monitor this, if the duplicate files are many e.g themes map distribution files, we should update this to batch delete files since the API supports batch requests.

Reply:
I'm not too worried about the performance of error recovery, especially from my anecdotal experience of how infrequently this happens, but you're right, we can monitor and be prepared to improve if needed.